### ISSUE TITLE
Undeclared dependency Type::Nano

### ISSUE THREAD

**Issue by eserte** [kfly8/Variable-Declaration#6]
It seems that Type::Nano has to be declared as a dependency, otherwise its absence would cause test failures:
```
...
Can't locate Type/Nano.pm in @INC (you may need to install the Type::Nano module) (@INC contains: /home/cpansand/.cpan/build/2019010114/Variable-Declaration-0.02-2/blib/arch /home/cpansand/.cpan/build/2019010114/Variable-Declaration-0.02-2/blib/lib /opt/perl-5.26.2t/lib/site_perl/5.26.2/x86_64-linux-thread-multi /opt/perl-5.26.2t/lib/site_perl/5.26.2 /opt/perl-5.26.2t/lib/5.26.2/x86_64-linux-thread-multi /opt/perl-5.26.2t/lib/5.26.2 .) at t/03_level/01_level_0.t line 5.
BEGIN failed--compilation aborted at t/03_level/01_level_0.t line 5.
t/03_level/01_level_0.t .... 
Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
... (etc) ...
```

**Comment by kfly8** [comment]
Thank you for the test report.

Type::Nano is bundled with Type::Tie 0.010.
So I specified this version.

**Comment by kfly8** [comment]
I released a version 0.03 now.
So check this version.

Thanks!
